src/worker/wtm-privacy-score.js

```diff
@@ -0,0 +1,41 @@
rten.pt":[11,1,4,0,4,2,1,1,2,0,2,0,0],"swtor.com":[2,1,3,0,0,4,1,0,0,0,2,0,0],"rarbgdata.org":[0,0,0,0,0,0,1,0,0,0,0,0,2],"wayfair.co.uk":[9,0,5,0,1,1,1,1,1,0,5,1,1],"medlineplus.gov":[0,0,1,0,0,1,1,0,0,0,4,0,0],"containerstore.com":[17,1,7,0,2,2,0,0,2,0,7,1,0],"nitori-net.jp":[12,1,5,0,0,2,1,0,1,0,5,2,0],"digitalcombatsimulator.com":[3,1,1,0,0,1,1,0,0,0,2,0,0],"sheridancollege.ca":[2,1,5,0,2,2,1,1,1,0,3,0,0],"msn.cn":[15,0,3,0,0,0,1,0,1,0,1,1,0],"cedeo.fr":[4,0,5,0,0,2,1,0,0,0,6,1,0],"asianwiki.com":[8,0,4,0,1,1,1,1,0,0,0,2,0],"bnc.ca":[5,1,5,0,2,2,1,1,0,0,4,0,0],"morazzia.com":[1,0,2,0,0,1,1,0,0,1,1,0,0],"tuttomercatoweb.com":[10,0,2,0,0,2,1,1,0,0,1,2,0],"wantedly.com":[7,1,5,0,1,1,0,2,0,0,6,3,0],"nfieldmr.com":[0,0,4,0,0,0,1,0,0,0,1,0,0],"lecker.de":[29,1,5,0,1,2,1,1,1,0,3,1,0],"commentcamarche.com":[21,0,4,0,2,1,1,0,0,0,3,1,0],"zalando-lounge.de":[4,0,4,0,3,1,1,1,1,0,1,0,0],"augsburger-allgemeine.de":[27,0,4,0,1,2,1,2,3,0,4,0,0],"123moviesfree.net":[3,1,4,0,0,1,2,1,0,0,3,0,0],"thehindu.com":[15,2,6,0,0,5,1,1,0,0,6,5,1],"weissratings.com":[1,1,4,0,0,1,0,0,2,0,3,0,0],"argentina.gob.ar":[3,1,7,0,0,1,1,1,0,0,1,2,0],"zimmo.be":[23,0,7,0,1,2,0,1,0,0,4,0,0],"secureinternetbank.com":[0,0,1,0,0,3,1,1,0,0,1,0,0],"flexpool.io":[0,0,1,0,3,1,1,0,1,0,1,0,0],"betterhelp.com":[9,0,4,0,2,3,0,2,0,0,3,6,0],"escortsclub.gr":[0,0,2,0,0,1,0,0,0,0,0,0,0],"ebalka.fun":[1,0,2,0,0,0,1,0,0,0,1,0,0],"pdf24.org":[4,0,4,0,1,1,1,0,0,0,1,0,0],"macro.com.ar":[4,0,3,0,1,2,1,0,0,0,3,0,0],"skribbl.io":[8,0,5,0,0,1,1,0,0,0,1,0,0],"getchu.com":[5,1,6,0,1,0,1,0,1,0,1,2,0],"si.edu":[9,2,10,1,2,3,0,2,3,0,5,4,0],"horoscope.com":[16,0,6,0,0,3,1,0,1,0,5,2,0],"groupon.fr":[9,1,4,0,1,2,1,1,0,0,3,1,0],"b-ok.xyz":[0,0,4,1,0,0,1,0,0,0,0,0,0],"evite.com":[27,0,7,0,2,2,0,2,1,0,7,3,0],"chaoxing.com":[1,0,4,0,0,0,1,0,1,0,0,0,0],"goldtokens.net":[2,0,6,0,2,2,0,0,0,0,1,2,0],"videotron.com":[8,4,12,0,2,3,1,2,0,0,6,2,1],"theroot.com":[22,1,6,0,0,2,0,0,1,0,6,2,2],"sxypix.com":[0,0,1,0,0,1,2,0,0,0,0,0,0],"ntu.edu.sg":[2,2,6,0,0,2,1,0,2,0,3,0,0],"boursier.com":[17,0,2,0,0,3,1,0,0,0,4,0,0],"ilcorsaronero.link":[2,0,4,0,0,1,1,0,1,0,1,0,1],"thefarside.com":[14,0,2,1,0,2,0,0,0,0,4,0,0],"sinonimos.com.br":[2,0,5,0,0,1,1,0,0,0,4,0,0],"zalando.no":[4,0,3,0,1,1,0,1,1,0,1,0,0],"mcmaster.com":[0,0,2,0,1,0,0,0,0,0,0,0,0],"tezfiles.com":[1,0,3,0,0,1,1,0,0,0,2,0,0],"androidpolice.com":[39,2,5,1,1,1,0,1,1,0,3,2,0],"seterra.com":[25,1,5,0,0,1,1,0,0,0,2,0,0],"wallhere.com":[2,0,2,0,0,0,1,0,0,0,1,0,0],"mejortorrentes.com":[1,0,3,0,0,1,1,0,0,0,1,0,1],"sonos.com":[13,1,6,0,4,5,1,2,0,0,8,2,1],"petitesannonces.ch":[2,0,0,0,0,0,1,0,0,0,1,0,0],"ursa-tm.ru":[3,1,7,0,0,1,1,0,0,0,3,5,0],"pcgarage.ro":[5,2,3,0,1,1,1,0,0,0,1,1,1],"streamdungeons.com":[0,1,8,0,2,1,1,0,1,0,2,0,0],"yourdictionary.com":[18,0,3,0,1,3,1,1,0,0,2,1,0],"vipboxtv.se":[4,1,2,0,1,1,1,1,0,0,1,0,3],"webfail.com":[53,1,8,0,2,4,1,1,1,0,1,2,0],"cbs.com":[29,0,9,0,1,4,1,0,2,0,10,0,0],"google.co.il":[1,1,5,0,0,0,0,1,0,0,1,0,0],"uworld.com":[4,1,6,0,1,2,1,1,1,0,5,2
```

Comment:
categories should already by loaded in `storage`

Reply:
That file is from the generator (other project). I would recommend to only use "lookupWtmPrivacyScoreForSite" in this file, the request is implementation detail.

It should match the categories that we have though. The order is not guaranteed to be the same.